### ISSUE TITLE
stem channels and downsample

### ISSUE THREAD

**Issue by JingweiZhang12** [NoamRosenberg/AutoML#27]
Hi, thanks for your codes!
In the original paper, 'every blue node in Fig.1 with downsample rate s has B x Fx s/4' output filters. In your codes,  the  stems' channels setting are as following:

self.stem1 = nn.Sequential(
            nn.Conv2d(half_f_initial* self._block_multiplier, half_f_initial* self._block_multiplier, 3, stride=1, padding=1),
            nn.BatchNorm2d(half_f_initial* self._block_multiplier),
            nn.ReLU ()
        )
self.stem2 = nn.Sequential(
            nn.Conv2d(half_f_initial* self._block_multiplier, f_initial* self._block_multiplier, 3, stride=2, padding=1),
            nn.BatchNorm2d(f_initial* self._block_multiplier),
            nn.ReLU ()
        )
I want to know why the stem1 and stem2 channels are  half_f_initial* self._block_multiplie and   f_initial* self._block_multiplier respectively rather than f_initial* self._block_multiplier and  filter_multiplier* self._block_multiplier? I think latter is more in line with the original paper.

**Comment by NoamRosenberg** [comment]
Hi @JingweiZhang12 , thanks for reading the code. Definitely helps to have another pair of eyes on this. Originally we consulted the author and he told us 10 and 20 filters, but I think he was mistaken. You are right, it does fit the paper better, and @zhizhangxian has already tested this and it’s showing better results so far. So following your advice I have merged a fix. 

Keep reading the code, and if you find things you think are better, I would be very pleased to have you join in and contribute to the project. All the best,
